metadata-models/src/main/pegasus/com/linkedin/ml/metadata/MLModelProperties.pdl

```diff
@@ -0,0 +1,46 @@
+namespace com.linkedin.ml.metadata
+
+import com.linkedin.common.MLFeatureUrn
+import com.linkedin.common.Time
+import com.linkedin.common.VersionTag
+
+/**
+ * Properties associated with a ML Model
+ */
+record MLModelProperties {
+
+  /**
+   * Documentation of the MLModel
+   */
+  description: optional string
+
+  /**
+   * Date when the MLModel was developed
+   */
+  date: optional Time
+
+  /**
+   * Version of the MLModel
+   */
+  version: optional VersionTag
+
+  /**
+   * Type of Algorithm or MLModel such as whether it is a Naive Bayes classifier, Convolutional Neural Network, etc
+   */
+  type: optional string
+
+  /**
+   * Hyper Parameters of the MLModel
+   */
+  hyperParameters: optional array[string]
```

Comment:
I think `hyperParameters` should be represented as a map of parameter name and associated value i.e. a map whose key is of type string, and value is typeref union of int, string, bool, etc https://en.wikipedia.org/wiki/Hyperparameter_(machine_learning). I could be mistaken, would love to hear your thoughts.

Reply:
I got myself educated on `hyperparameters`, I agree! Will update the same.